pypeit/wavecalib.py

```diff
@@ -779,6 +789,10 @@ def redo_echelle_orders(self, bad_orders:np.ndarray, dets:np.ndarray, order_dets
             dets (np.ndarray): detectors of the spectrograph
                 Multiple numbers for mosaic (typically)
             order_dets (np.ndarray): Orders on the each detector
+            bad_orders_maxfrac (float): Maximum fraction of bad orders
+            in a detector for attempting a refit
+            frac_rms_thresh (float): Fractional change in the RMS threshold
+            for accepting a refit
```

Comment:
I think lines need to be indented for parameter definitions longer than one line.  i.e.:
```
bad_orders_maxfrac (float): Maximum fraction of bad orders
    in a detector for attempting a refit
frac_rms_thresh (float): Fractional change in the RMS threshold
    for accepting a refit
```

Reply:
Done